### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by TranVanDu**

### CODE REVIEW

**Comment by TranVanDu** [comment]
ready

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+
+# Common configuration.
+AllCops:
+  # Include gemspec and Rakefile
+  Include:
+    - '**/*.gemspec'
+  Exclude:
+    - 'vendor/**/*'
+    - 'config/unicorn.rb'
+    - 'bin/*'
```
Comment (thanhhungchu95) [inline .rubocop.yml]: thêm 2 dòng
```
- 'config.ru'
- 'Rakefile'

**Comment by TranVanDu** [comment]
image

**Comment by TranVanDu** [comment]
ready

**Comment by DuongNT1512** [comment]
2 commit r nhé

**Comment by TranVanDu** [comment]
ready

**Comment by TranVanDu** [comment]
ready

**Comment by TranVanDu** [comment]
image

**Comment by TranVanDu** [comment]
image

**Comment by TranVanDu** [comment]
image

**Comment by TranVanDu** [comment]
ready

**Pull request merged**